### PR TITLE
Tomm/e2e ramps

### PR DESCRIPTION
**Pull request by MuckT**
### Description

Increases the automated coverage of our on and off ramps in the Valora. Tests focus on in app behavior along with checking integration with 3rd party providers. PNG snapshots are used to confirm that 3rd navigating away from the app behaves as expected.

### Other changes

- [x] Updates Jest Snapshots with new testID's.
- [x] Add associated png snapshots for Android and iOS.

### Tested

- [x] All tests running locally on iOS and Android devices matching Circle Ci Configurations
- [x] Run in Circle Ci 
- [x] Individual specs run locally

### How others should test

* Navigate to the `wallet/packages/mobiles` directory.

* All Tests can be run

  ```sh
  # Using Multiple Emulated Devices
  yarn test:e2e:ios -w 4
  
  # Runs One Emulated Device
  yarn test:android -w  1
  ```

* A Specific spec.js can be run 

  ```sh
  yarn test:e2e:ios -w 1 -f OnRamps.spec.js
  ```

* Rerunning with out rebuilding

  ```sh
  # Both 'OnRamps.spec.js' and 'OffRamps.spec.js' will run
  yarn test:e2e:ios -w 2 -d -f Ramps.spec.js
  ```

This does not need to be tested by QA, but monitoring build times prior to merge to main will be important as the asset folder grows.

### Backwards compatibility

This PR can be merged alone and also contains all of the contents of PR #260.

### CODE REVIEW

File: packages/mobile/e2e/src/usecases/Send.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ export default Send = () => {
     // Look for the latest transaction and assert
     await waitFor(element(by.text(`${randomContent}`)))
       .toBeVisible()
-      .withTimeout(45000)
+      .withTimeout(90000)
```
Comment (etuleu) [inline packages/mobile/e2e/src/usecases/Send.js:64]: was this failing? or why do we need more time?

File: packages/mobile/e2e/src/utils/utils.js [hunk 2]
```diff
@@ -251,3 +251,11 @@ export async function scrollIntoView(scrollTo, scrollIn, speed = 350, direction
       .scroll(speed, direction)
   } catch {}
 }
+
+export async function getDeviceModel() {
```
Comment (etuleu) [inline packages/mobile/e2e/src/utils/utils.js:255]: I thought there was a detox util that does this but I might be mistaking it for something else ...

**Review by etuleu — APPROVED** [review]
This is awesome!

File: packages/mobile/e2e/src/usecases/Send.js (hunk 1 above)
Comment (MuckT) [reply packages/mobile/e2e/src/usecases/Send.js:64]: The test was failing in the specified timeout in Circle Ci. I added additional allowed timeout and it started passing. This timeout will take up to 90 seconds; however, if the text is visible before during that time it'll pass and continue.

File: packages/mobile/e2e/src/utils/utils.js (hunk 2 above)
Comment (MuckT) [reply packages/mobile/e2e/src/utils/utils.js:255]: There is a `device.getPlatform()`, but no model information is provided. `device.getUiDevice()` is Android only at present.

https://github.com/wix/Detox/blob/master/docs/APIRef.DeviceObjectAPI.md

**Pull request merged**